R/tm_g_gh_correlationplot.R

```diff
@@ -868,6 +879,41 @@ srv_g_correlationplot <- function(id,
       brushing = TRUE
     )
 
+
+    ### REPORTER
+    if (with_reporter) {
+      card_fun <- function(comment) {
+        card <- teal.reporter::TealReportCard$new()
+        card$set_name("Correlation plot")
+        card$append_text("Correlation plot", "header2")
+        card$append_text("Filter State", "header3")
+        card$append_fs(datasets$get_filter_state())
+        card$append_text("Data constraint", "header3")
+        card$append_text(
+          formatted_data_constraint(input$constraint_var, input$constraint_range_min, input$constraint_range_max)
+        )
+        card$append_text("Correlation plot", "header3")
```

Comment:
```suggestion
        card$append_text("Plot", "header3")
```

We like your suggestion please use the Plot/Table for each subtitle for the main element.